### PR TITLE
IA REFRESH - (DOCSP-26919) [Port] Copy an App (/apps/copy)

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-26919) [Port] Copy an App (/apps/copy)

### Staged Changes

- [Copy an App](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/apps/copy/)

### CODE REVIEW

File: source/apps/copy.txt [hunk 1]
```diff
@@ -0,0 +1,200 @@
+.. _copy-app:
```
Comment (mongodben) [inline source/apps/copy.txt:1]: is this just move of `configure/copy-app.txt`?

**Comment by mongodben** [comment]
higher lvl question for the Develop and Deploy Apps section IA changes? 

i notice that the [Create page](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/apps/create/) lists the different admin interfaces in sections in order, while the other pages that i've been reviewing use tabs. is this diff intentional?

File: source/apps/copy.txt [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can make a copy of an existing App by reusing the App's
+configuration files and manually porting other information like Secrets.
```
Comment (mongodben) [inline source/apps/copy.txt:17]: here it says "and manually porting other information like Secrets". what other info besides secrets would you have to manually port?

File: source/apps/copy.txt [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+You might want to copy an application if:
+
+- You use feature branches for development. Use a unique copy of the
+  App for each feature branch to avoid conflicts.
+
+- You run tests against a working version of the App. You can copy an
+  App for each test run to to ensure a consistent start state.
+
+- You deploy the same app to clusters across regions using a :ref:`local
+  deployment model <local-deployment>`. You can copy an App to servce
```
Comment (mongodben) [inline source/apps/copy.txt:28]:
```suggestion
  deployment model <local-deployment>`. You can copy an App to service
```

typo fix. also think 'service' could be replaced by better word. 'serve' maybe?

File: source/apps/copy.txt [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+      .. procedure::
+
+         .. step:: Authenticate a MongoDB Atlas User
+
+            .. include:: /includes/cli-login.rst
+
+         .. step:: Create a New App
+
+            Create a new blank App. Choose the same name as the original
+            App and set the deployment model and region to be what you
```
Comment (mongodben) [inline source/apps/copy.txt:73]: why must name be the same?

File: source/apps/copy.txt [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+   .. tab::
+      :tabid: cli
+
+      You will need the following to copy an App in the CLI:
+
+      - .. include:: /includes/prereqs/atlas-account.rst
+
+      - .. include:: /includes/prereqs/atlas-admin-api-key.rst
+
+      - .. include:: /includes/prereqs/app-services-cli.rst
```
Comment (mongodben) [inline source/apps/copy.txt:45]: not part of this PR so can't touch but this sentence reads funky to me:

> A [properly installed](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/cli/#std-label-install-realm-cli) copy of [realm-cli](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/cli/#mongodb-binary-bin.realm-cli) that has been added to your system PATH.

perhaps refactor to smthn like: 

"realm-cli installed and added to your system PATH."

**Review by mongodben — APPROVED** [review]
lgtm w a few comments

File: source/apps/copy.txt (hunk 1 above)
Comment (nlarew) [reply source/apps/copy.txt:1]: Not just a move, I also updated the procedures

**Comment by nlarew** [comment]
@mongodben 
> higher lvl question for the Develop and Deploy Apps section IA changes.
>
> i notice that the [Create page](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/ia-apps-copy/apps/create/) lists the different admin interfaces in sections in order, while the other pages that i've been reviewing use tabs. is this diff intentional?

Yeah intentional for now but I plan to change it to match these other pages. I changed my mind on it halfway through the section.

File: source/apps/copy.txt (hunk 4 above)
Comment (nlarew) [reply source/apps/copy.txt:73]: good point - can copy the config to an app with a different name

File: source/apps/copy.txt (hunk 2 above)
Comment (nlarew) [reply source/apps/copy.txt:17]: I was thinking users when I wrote this but there's not really a way to port users so I'll update here
